cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,40 @@
+describe('User login functionality', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.wait(2000)
+  })
+
+  const openLoginForm = () => {
```

Comment:
I see here that you close the register form and open a login intead of clicking the login-btn in the registerform. I think most people would click the login btn in the registerform. Maybe test this out? I had trouble with this one myself, and figured i had to wait() after clicking the btn to get the modal to load. 

ex: 
cy.get("#registerForm")
    .should("be.visible")
    .parent()
    .find('[data-auth="login"]')
    .click();
  cy.wait(500);